pkg/nydussdk/client.go

```diff
@@ -29,6 +30,9 @@ const (
 	endpointMount      = "/api/v1/mount"
 	endpointMetrics    = "/api/v1/metrics"
 	endpointBlobs      = "/api/v2/blobs"
+	endpointTakeOver   = "/api/v1/daemon/fuse/takeover"
```

Comment:
Maybe better to explain the three new APIs.

Reply:
fixed